pyproject.toml

```diff
@@ -24,7 +24,6 @@ requests = "^2.28.2"
 escapism = "^1.0.1"
 semver = "^2"
 rpaframework = "^27.2.0"
-yq = "^3.1.0"
```

Comment:
Out of my curiosity - what was the particular problem here if you can describe a bit?

Reply:
it was complaining for `-i (--inplace)`. In the python version of the library it must be `Y --in-place`.

Reply:
So the very same incompatibility we already experienced. Thanks for clarification.

Reply:
yep